molecule/default/molecule.yml

```diff
@@ -0,0 +1,79 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+lint: |
+  set -e
+#  yamllint .
+#  ansible-lint
+platforms:
+#--- postgresql 9.5 no longer supported, https://www.postgresql.org/support/versioning/
```

Comment:
If we don't intend to support it we can just pull the comment block.

Reply:
I would like to use them quite soon, but that is a different work package that requires a different discussion about code guidelines.

Reply:
Agree, removing 9.5